aws/resource_aws_gamelift_fleet_test.go

```diff
@@ -0,0 +1,625 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/gamelift"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+const testAccGameliftFleetPrefix = "tf_acc_fleet_"
+
+func init() {
+	resource.AddTestSweepers("aws_gamelift_fleet", &resource.Sweeper{
+		Name: "aws_gamelift_fleet",
+		Dependencies: []string{
+			"aws_gamelift_build",
+		},
+		F: testSweepGameliftFleets,
+	})
+}
+
+func testSweepGameliftFleets(region string) error {
+	client, err := sharedClientForRegion(region)
+	if err != nil {
+		return fmt.Errorf("error getting client: %s", err)
+	}
+	conn := client.(*AWSClient).gameliftconn
+
+	resp, err := conn.ListFleets(&gamelift.ListFleetsInput{})
+	if err != nil {
+		return fmt.Errorf("Error listing Gamelift Fleets: %s", err)
+	}
+
+	if len(resp.FleetIds) == 0 {
+		log.Print("[DEBUG] No Gamelift Fleets to sweep")
+		return nil
+	}
+
+	out, err := conn.DescribeFleetAttributes(&gamelift.DescribeFleetAttributesInput{
+		FleetIds: resp.FleetIds,
+	})
+	if err != nil {
+		return fmt.Errorf("Error describing Gamelift Fleet attributes: %s", err)
+	}
+
+	log.Printf("[INFO] Found %d Gamelift Fleets", len(out.FleetAttributes))
+
+	for _, attr := range out.FleetAttributes {
+		if !strings.HasPrefix(*attr.Name, testAccGameliftFleetPrefix) {
+			continue
+		}
+
+		log.Printf("[INFO] Deleting Gamelift Fleet %q", *attr.FleetId)
+		err := resource.Retry(1*time.Minute, func() *resource.RetryError {
```

Comment:
Does this need to be 60 minutes to match the delete function? Otherwise the sweeper could trigger CI failures consistently.